loguru/_logger.py

```diff
@@ -890,11 +890,11 @@ def add(
                 elif isinstance(level_, str):
                     try:
                         levelno_ = self.level(level_).no
-                    except ValueError:
+                    except ValueError as err:
                         raise ValueError(
                             "The filter dict contains a module '%s' associated to a level name "
                             "which does not exist: '%s'" % (module, level_)
-                        )
+                        ) from err
```

Comment:
Can we just `raise` from `None` here please? It's consistent with other raised `Exception` in the module. I prefer to hide implementation details from the stack traceback.

Reply:
Makes sense. Done.